src/foraging_gui/TransferToNWB.py

```diff
@@ -38,8 +38,17 @@ def _get_field(obj, field_list, reject_list=[], index=None, default=np.nan):
         field_list = [field_list]
         
     for f in field_list:
-        if hasattr(obj, f):
-            value = getattr(obj, f)
+            tag=0
```

Comment:
Can you use more readable variable names?

Reply:
I changed the name to `has_field`. 

Hi Han, thanks for reviewing this over the weekend. I don't expect you to respond so quickly.